src/middleware/express/login-success.js

```diff
@@ -4,7 +4,7 @@
 
 export default function loginSuccess(options = {}) {
   return function(req, res, next) {
-    if (options.successRedirect !== undefined) {      
+    if (options.successRedirect !== undefined && options.successRedirect !== null) {      
```

Comment:
It should probably just be `if (options.successRedirect) {...}`. I don't know why I didn't do that. Might have been a copy-paste remnant from how we were handling redirects pre 0.8. That makes more sense I think.